### PR TITLE
Additional Dockerfile for self-hosting

### PR DESCRIPTION
**Pull request by michielbdejong**
The existing Dockerfile in the repo root is meant for local development. This additional Dockerfile (not sure if the docs/ folder would be the right location) is meant for use by the self-hosting guide I'm working on (currently at https://github.com/michielbdejong/fxa-self-hosting).

### CODE REVIEW

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 62301f4c4f1a2e395c31d6dc308df63996c51e43
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 62301f4c4f1a2e395c31d6dc308df63996c51e43
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 11931d26eb278164f17d3386958d87722a657a4e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 62301f4c4f1a2e395c31d6dc308df63996c51e43
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 11931d26eb278164f17d3386958d87722a657a4e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1c31a663bee1220df7c78fe65a7fc447c55d3e95
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were the following style issues with your Pull Request:

 * Commit: 62301f4c4f1a2e395c31d6dc308df63996c51e43
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 11931d26eb278164f17d3386958d87722a657a4e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 1c31a663bee1220df7c78fe65a7fc447c55d3e95
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 2dcea80ae9a8ca17b1d87759b4f698e1d2d718ed
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-oauth-server

---
This message was auto-generated by https://gitcop.com

File: docs/self-host.docker [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM vladikoff/fxa-slim-image:latest
```
Comment (vladikoff) [inline docs/self-host.docker]: let's lock this version to 1.0.0 @michielbdejong

Comment (michielbdejong) [inline docs/self-host.docker]: Thanks! Done.

**Pull request merged**